types/fs-extra/fs-extra-tests.ts

```diff
@@ -141,6 +141,14 @@ fs.remove(dir, errorCallback);
 fs.remove(dir).then(() => {
     // stub
 });
+// @ts-expect-error map can't be called as it passes a number for the second argument instead of a callback.
+["file/to/remove"].map(fs.remove);
+
+// @ts-expect-error promise should not be returned when callback is provided.
+// tslint:disable-next-line:no-void-expression
+fs.remove(dir, errorCallback).then(() => {
+	// stub
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/fs-extra/fs-extra-tests.ts;
  Line: 150;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.